### PR TITLE
Update info on free transactions

### LINKED ISSUE #29363: Misguided information on Here Travel Time page
### Feedback

This page is referred to an old statement of here maps. It is said on the page that here maps has 250.000 free transactions but normally this number is only for 5000. The page should immediately be updated with the new info in order to prevent unwanted usafe and bills. 

### URL

https://www.home-assistant.io/integrations/here_travel_time/

### Version

2023.10.3

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by eifinger**
## Proposed change

Updates the links to the new HERE platform documentation as well as the information on the changed free transactions.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #29363

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/here_travel_time.markdown [hunk 1]
```diff
@@ -14,15 +14,13 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The `here_travel_time` sensor provides travel time from the [HERE Routing API](https://developer.here.com/documentation/routing-api/dev_guide/index.html).
+The `here_travel_time` sensor provides travel time from the [HERE Routing API](https://www.here.com/docs/bundle/routing-api-developer-guide-v8/page/README.html).
 
 ## Setup
 
-You need to register for an API key (REST & XYZ HUB API/CLI) by following the instructions [here](https://developer.here.com/tutorials/getting-here-credentials/).
+You need to register for an API key by following the instructions [here](https://www.here.com/docs/bundle/routing-api-developer-guide-v8/page/topics/send-request.html).
```
Comment (c0ffeeca7) [inline source/_integrations/here_travel_time.markdown]:
```suggestion
You need to register for an API key by following the instructions in the [API Developer Guide](https://www.here.com/docs/bundle/routing-api-developer-guide-v8/page/topics/send-request.html).
```

File: source/_integrations/here_travel_time.markdown [hunk 2]
```diff
@@ -14,15 +14,13 @@ ha_platforms:
...
 
-HERE offers a Freemium Plan which includes 250,000 free Transactions per month. For the Routing API, one transaction equals one request with one starting point (no multi stop). If you are not [updating sensors on demand](#updating-sensors-on-demand-using-automation) you can track 28 routes without exceeding the limit. More information can be found [here](https://knowledge.here.com/csm_kb?id=public_kb_csm_details&number=KB0016433)
-
-By default HERE will deactivate your account if you exceed the free Transaction limit for the month. You can add payment details to re-enable your account as described [here](https://knowledge.here.com/csm_kb?id=public_kb_csm_details&number=KB0016434)
+HERE offers a Limited Plan which includes 1000 free transactions per day. If you are not [updating sensors on demand](#updating-sensors-on-demand-using-automation) you can track 3 routes without exceeding the limit. You can provide payment details to increase this to 5000 free transactions per day. More information can be found [here](https://www.here.com/get-started/pricing)
```
Comment (c0ffeeca7) [inline source/_integrations/here_travel_time.markdown]:
```suggestion
HERE offers a Limited Plan which includes 1000 free transactions per day. If you are not [updating sensors on demand](#updating-sensors-on-demand-using-automation), you can track 3 routes without exceeding the limit. You can provide payment details to increase this to 5000 free transactions per day. More information can be found [on the pricing page](https://www.here.com/get-started/pricing)
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @eifinger :+1:

**Comment by atfalk** [comment]
Much appreciate it! @eifinger

**Pull request merged**